spec/models/service_spec.rb

```diff
@@ -133,7 +133,7 @@
 
       it "returns the uniq value for the 'off' power state" do
         expect(@service).to receive(:map_power_states).with(:stop).and_return(["off"])
-        expect(@service).to receive(:vm_power_states).and_return(["off"])
+        expect(@service).to receive(:power_states).and_return(["off"])
```

Comment:
@syncrou Is it possible to write this test with actual services and power states instead of stubbing out everything? (if we were to change the query from an N+1 to a single query, it would make the testing much easier